### ISSUE TITLE
Failing to read mesages with -c on first mesage, with an old config

### ISSUE THREAD

**Issue by syonfox** [drorm/gish#17]
```
sudo npm install -g gish-gpt

gish -c .. hi 


file:///usr/lib/node_modules/gish-gpt/dist/gptRequest.js:101
        messages.push({ role: "user", content: request });
                 ^

TypeError: Cannot read properties of undefined (reading 'push')
    at GptRequest.fetch (file:///usr/lib/node_modules/gish-gpt/dist/gptRequest.js:101:18)
    at GptRequest.submitChat (file:///usr/lib/node_modules/gish-gpt/dist/gptRequest.js:64:37)
    at Interactive.submitChat (file:///usr/lib/node_modules/gish-gpt/dist/interactive.js:120:26)
    at Interactive.handlCommand (file:///usr/lib/node_modules/gish-gpt/dist/interactive.js:114:28)
    at Interface.<anonymous> (file:///usr/lib/node_modules/gish-gpt/dist/interactive.js:76:24)
    at Interface.emit (node:events:513:28)
    at [_onLine] [as _onLine] (node:internal/readline/interface:422:12)
    at [_line] [as _line] (node:internal/readline/interface:893:18)
    at [_ttyWrite] [as _ttyWrite] (node:internal/readline/interface:1271:22)
    at ReadStream.onkeypress (node:internal/readline/interface:270:20)

Node.js v18.15.0

```

https://github.com/drorm/gish/blob/6ec8161522741dd816cc29cec4bf1a0f068a5482/src/gptRequest.ts#L111

looks like we just need to add a check here if the previous message is not found.
```
if(!Array.isArray(messages)) {
  console.warn("Unable to process previous message is this a new chat? is history corrupt?")
  this.messages = [];
}

```

defiantly a better way of implementing but would work.

this might only be a problem on initial migration from an older version

**Comment by drorm** [comment]
The price of laziness.
As I was writing the code I thought there's very little chance that someone would run into this :-), and sure enough ..
I'll go ahead and fix it in 0.4 in conjunction with #13

**Comment by drorm** [comment]
Looking at this some more, there are too many failure modes that would need to be handled: corrupted JSON file, non-existent, etc. 
Rather than try to find all the edge cases and have a nice pretty error message, I'd rather have the exceptions at this point, which make it easy to analyze.
